xarray/tests/test_conventions.py

```diff
@@ -274,3 +274,6 @@ def test_invalid_dataarray_names_raise(self):
 
     def test_encoding_kwarg(self):
         pass
+
+    def test_encoding_kwarg_fixed_width_string(self):
+        pass
```

Comment:
I'm not sure I understand the purpose of this...

Reply:
This `TestCFEncodedDataStore` test is intended for checking the backend interface without depending on any particular file format. That means it doesn't support string encodings. (Added a comment.)